src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -32,7 +32,7 @@ public class InputTagHelper : TagHelper
                 { "Url", "url" },
                 { "EmailAddress", "email" },
                 { "Date", "date" },
-                { "DateTime", "datetime" },
+                { "DateTime", "datetime-local" },
```

Comment:
@dougbu So the change in `TemplateRenderer` [lines 208-210](https://github.com/aspnet/Mvc/blob/48240d4b89d5f04a68bb8a5737431405b46dc415/src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/TemplateRenderer.cs#L208-L210) to return "datetime-local" will cover (3)?